backend/blueprint/api/admin/users.py

```diff
@@ -0,0 +1,137 @@
+from flask import Blueprint, jsonify, request
+from api.utils.response import Response
+from firebase_admin import auth
+
+
+router = Blueprint('users', __name__)
+
+
+@router.route('/createuser', methods=['POST'])
+def create_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user = auth.create_user(
+            display_name=data['name'],
+            email=data['email'],
+            email_verified = data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled = data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully created new user named {0} with user-id: {1}'.format(user.display_name, user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        print(e)
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/updateuser', methods=['POST'])
+def update_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user = auth.update_user(
+            data['uid'],
+            display_name=data['name'],
+            email=data['email'],
+            email_verified = data['email_verified'],
+            photo_url=data['photo_url'],
+            password=data['password'],
+            disabled = data['disabled']
+        )
+        return jsonify({
+            'message': 'Sucessfully updated user: {0}'.format(user.uid),
+            'status': 200
+        })
+    except Exception as e:
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/deleteuser', methods=['POST'])
+def delete_user():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        auth.delete_user(
+            data['uid']
+        )
+        return jsonify({
+            'message': 'Successfully deleted user',
+            'status': 200
+        })
+    except Exception as e:
+        return jsonify({
+            'message': str(e),
+            'status': 400
+        })
+
+
+@router.route('/userdetails', methods=['POST'])
+def get_user_details():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(Response(500).generateMessage(str(e)))
+        
+    try:
+        user_details = auth.get_user(
+            data['uid']
+        )
+        name = user_details.display_name
+        email = [{'address': user_details.email, 'verified': user_details.email_verified}]
+        photo_url = user_details.photo_url
```

Comment:
Have u worked on the review on #779 . It seems that you are just ignoring the review and only copy pasting the code from the documentation. This is not a good practice as this doesn't show you work. Please try to respect our review and do the suggested changes. There is no attribute as such in our db as `photo_url`